services/src/main/java/org/exoplatform/news/NewsServiceImpl.java

```diff
@@ -960,6 +973,10 @@ public News createNewsDraft(News news) throws Exception {
     newsDraftNode.setProperty("exo:pinned", false);
     newsDraftNode.setProperty("exo:archived", false);
     newsDraftNode.setProperty("exo:spaceId", news.getSpaceId());
+    if (newsDraftNode.canAddMixin(MIX_NEWS_DRAFT_VISIBILITY)) {
+      newsDraftNode.addMixin(MIX_NEWS_DRAFT_VISIBILITY);
+      newsDraftNode.setProperty(MIX_NEWS_DRAFT_VISIBILITY_PROP, String.valueOf(news.isDraftVisible()));
```

Comment:
mixin should be added only once
```suggestion
if (newsDraftNode.canAddMixin(MIX_NEWS_DRAFT_VISIBILITY) && !newsDraftNode.hasProperty(MIX_NEWS_DRAFT_VISIBILITY_PROP)) {
  newsDraftNode.addMixin(MIX_NEWS_DRAFT_VISIBILITY);
}
newsDraftNode.setProperty(MIX_NEWS_DRAFT_VISIBILITY_PROP, String.valueOf(news.isDraftVisible()));
```